### DISCUSSION TITLE
Coexistence with Legacy IPv4 section

### DISCUSSION THREAD

**Discussion by EduardVasilenko** [becarpenter/book6#39] [General]
Hi Brian,
There is 3rd approach for IPv6 transition: slow migration host-by-host over one link.
It is perfectly discussed in the RIPE research: https://ripe85.ripe.net/wp-content/uploads/presentations/9-RIPE85-Deploying_IPv6_mostly.pdf
It is when the host claims by DHCPv4: I would support IPv6-only, help me with DNS64/NAT64 to reach the outdated Internet.
I propose to mention it.
Eduard

**Comment by EduardVasilenko** [comment]
this section did not have "Next" and "Previous" buttons.

**Comment by becarpenter** [comment]
Yes, it seems like we should mention RFC8925 and the RIPE presentation in the description of NAT64/DNS64.

That section needs a lot more text!

**Comment by becarpenter** [comment]
Closed as done
